sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Services/WebPubSubRequestExtensions.cs

```diff
@@ -152,11 +152,10 @@ internal static bool IsValidSignature(this WebPubSubConnectionContext connection
             {
                 var states = new Dictionary<string, object>();
                 var parsedStates = Encoding.UTF8.GetString(Convert.FromBase64String(connectionStates));
-                var statesObj = JsonDocument.Parse(parsedStates);
+                using var statesObj = JsonDocument.Parse(parsedStates);
```

Comment:
You don't have to convert `Convert.FromBase64String` to a string. JsonDocument.Parse takes a byte array.